.circleci/config.yml

```diff
@@ -45,7 +45,13 @@ jobs:
             - "/home/circleci/ffmpeg"
             - "/home/circleci/compiled"
           key: ffmpeg-cache-{{ checksum "install_ffmpeg.sh" }}
-
+          
+      - run:
+          name: "Download tasmodel.pb"
+          command: |
+            wget https://github.com/livepeer/livepeer-ml/releases/download/v0.3/tasmodel.pb
```

Comment:
Given `lpms` is not released separately, maybe not pin the version here and link to [latest](https://github.com/livepeer/livepeer-ml/releases/latest/download/tasmodel.pb) release, so we don't have to update CI job for every new version of the model?